torch_geometric/distributed/local_dataset.py

```diff
@@ -0,0 +1,216 @@
+from typing import Dict, List, Optional, Union
+
+import torch
+
+from torch_geometric.data import EdgeAttr, TensorAttr
+from torch_geometric.distributed.local_feature_store import \
+    LocalFeatureStore as Feature
+from torch_geometric.distributed.local_graph_store import \
+    LocalGraphStore as Graph
+from torch_geometric.typing import EdgeType, NodeType, TensorDataType
+from torch_geometric.utils import convert_to_tensor, squeeze
+
+
+class LocalDataset(object):
```

Comment:
Can we increase code coverage in this file? For me, it would be fine to drop support for `edge_features` completely for now, especially since `FeatureStore` doesn't support them yet anyway.

Reply:
Will try. :-) Thanks.